0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -236,6 +237,35 @@ p.send({
 
 For a detailed description of these properties, please see [`OutgoingTransfer`](#outgoingtransfer).
 
+#### sendMessage
+<code>ledgerPlugin.sendMessage( **message**:[OutgoingMessage](#outgoingmessage) ) ⇒ Promise.&lt;null></code>
```

Comment:
It would be good to include an explanation of what this feature is for. Also what should ledger plugin implementor do if the underlying ledger doesn't support messaging?